Source/src/WixSharp/Compiler.cs

```diff
@@ -2562,6 +2562,8 @@ static void ProcessProperties(Project wProject, XElement product, Dictionary<Fea
                                                .SetAttribute("Id", prop.Name)
                                                .SetAttribute("Value", prop.Value)
                                                .AddAttributes(prop.Attributes));
+                    if (prop.Secure.HasValue)

```

Comment:
You can use `.SetAttribute("Secure", prop.Secure)`.It handles `bool?` transparently.